### ISSUE TITLE
on change of height/width of parent container?

### ISSUE THREAD

**Issue by kumail-raza** [thomascharbit/jquery-fillcontainer#1]
can some how we could detect for the change of height/width of container? with no window.resize event being called?

**Comment by thomascharbit** [comment]
Hi,
AFAIK resize event only exists on window, not DOM elements :/

**Comment by kumail-raza** [comment]
@thomascharbit 
yes that what I asked is there a way to detect change on DOM elem, well there is upcoming HTML5 api for that https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver
